index.bs

```diff
@@ -20,7 +20,7 @@ Hello world!
 
 *This section is non-normative.*
 
-## Latency-sensitive applications in virtualized sessions ## {#browser-context-redirection}
+## Latency-sensitive applications in virtualized sessions ## {#browser-content-redirection}
```

Comment:
Thanks for catching this typo. I left an earlier comment in the last commit, then reviewed this commit and saw you'd fixed it.

Can you think of any reason not to just squash this commit into the last one in this PR before these land / were reviewed?